lib/helpers/s3-uploader.js

```
@@ -0,0 +1,74 @@
+	 * @param {string} contentS3Path - The path in the S3 bucket where the content will be uploaded.
+	 * @param {string} body - The content to be uploaded to the S3 bucket.
+	 * @returns {Promise<BucketInfo>} - Resolves with the bucket information of the successful upload.
+	 * @throws {SqsEmitterError} - Throws an error if the upload fails for all provided buckets.
+	 */
+	static async uploadContentS3Path(buckets, contentS3Path, body) {
+
+		const failedUploads = [];
+
+		for(const bucketInfo of buckets) {
+
+			const response = await this.uploadToBucket(bucketInfo, contentS3Path, body);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)